case-lib/hijack.sh

```diff
@@ -76,11 +76,11 @@ sudo()
             return $?
         ;;
         '1')    # sudo without passwd
-            eval $(echo "$SUDO_CMD $*")
+            eval $(echo "$SUDO_CMD env 'PATH=$PATH' $*")
             return $?
         ;;
         '2')    # sudo need passwd
-            eval $(echo "echo '$SUDO_PASSWD' | $SUDO_CMD -S $*")
+            eval $(echo "echo '$SUDO_PASSWD' | $SUDO_CMD -S env 'PATH=$PATH' $*")
```

Comment:
will `-E` help here to pass the ENV into sudo

Reply:
try with -E option which is not work in our script, because we are not `export` the ENV

Reply:
try with option `--preserve-env=PATH` it will report `command not found`

it seems is hijack effect